frontend/src/metabase/writeback/utils.ts

```diff
@@ -82,7 +82,9 @@ export function isMappedExplicitActionButton(
   dashCard: BaseDashboardOrderedCard,
 ): dashCard is ActionDashboardCard {
   const isAction = isActionDashCard(dashCard);
-  return isAction && typeof dashCard.action_id === "number";
+  return (
+    isAction && typeof dashCard.visualization_settings.action_slug === "string"
```

Comment:
here's another 😬 line. This should/will become a top level field on the backend, but for now it has to be a visualization setting.